cli/backup.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Democratized Data Foundation
```

Comment:
todo: Fix copyright year.

```suggestion
// Copyright 2023 Democratized Data Foundation
```